### ISSUE TITLE
the auto completion box doesn't disappear

### ISSUE THREAD

**Issue by AthenaProgram** [Microsoft/vscode#39980]
![image](https://user-images.githubusercontent.com/34213558/33804515-97ed3420-dde1-11e7-8c81-768ebc033b1d.png)
I'm new to vscode and writing javascript for cocos creator, in vim mode. The auto completion is nice but there always is a box hanging there, I pressed every key but it still doesn't disappear.
Any help would be greatful, thank you~

**Comment by mjbvz** [comment]
Does this only happen when using vim?

**Comment by AthenaProgram** [comment]
so sorry for the delay!
after your hint, I tried not using vim, I found every time I enter ',' , the box above would appear, and esc key could make it disappear;
But when using vim, esc is occupied by vim, then the box wouldn't disappear.

so how can I make the box disappear using vim?
![image](https://user-images.githubusercontent.com/34213558/34432462-61fa13c6-ecb2-11e7-846a-5d500a98aecb.png)
Actually I don't want the box everytime I pressed ',' could I close the function?
thank you so much!
